packages/arduino_ide.rb

```diff
@@ -19,15 +22,18 @@ class Arduino_ide < Package
   depends_on 'ant' => :build
   depends_on 'sommelier'
 
-  def self.build
     case ARCH
     when 'x86_64'
-      platform = 'linux64'
+      @platform = 'linux64'
     when 'i686'
-      platform = 'linux32'
+      @platform = 'linux32'
     when 'armv7l', 'aarch64'
-      platform = 'linuxarm'
+      @platform = 'linuxarm'
     end
+
+  def self.build
+    version = @version
+    platform = @platform
```

Comment:
These variable assignments shouldn't be necessary.  The instance variable defined at the top could be used anywhere in the class; including functions and methods.

Reply:
Done.